jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/LambdaRunConfigurationType.kt

```diff
@@ -20,10 +20,10 @@ class LambdaRunConfigurationType :
         AwsIcons.Resources.LAMBDA_FUNCTION
     ) {
     init {
-        // Although it should work, isApplicable doesn't seem to work for locallambdarunconfigurationfactory
-        // and it still shows up when it is not applicalbe. So we have to decide in the configuration to add it or not.
+        // Although it should work, isApplicable doesn't seem to work for LocalLambdaRunConfigurationFactory
+        // and it still shows up when it is not applicable. So we have to decide in the configuration to add it or not.
         // TODO see if this is resolvable
```

Comment:
Is this resolvable now that it's dynamic? :P

Reply:
Don't think so, they check the factory is applicable, if I remember right. Not the children in the factory.